content/en/docs/ambient/architecture/data-plane/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Ambient data plane
+description: Understand how the ambient data plane routes traffic between workloads in an ambient mesh.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
+1. **Out of Mesh**: a standard pod without any mesh features enabled. Istio and the ambient {{< gloss >}}data plane{{< /gloss >}} are not enabled.
+1. **In Mesh**: a pod that is included in the ambient {{< gloss >}}data plane{{< /gloss >}}, and has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/dataplane-mode=ambient` label. See [labels](docs/ambient/architecture#ambient-labels) for more details.
+1. **In Mesh, Waypoint enabled**: a pod that is "in mesh" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed. In this mode, L7 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/use-waypoint` label. See [labels](docs/ambient/architecture#ambient-labels) for more details.
+
+Depending on which category a workload is in, the traffic path will be different.
+
+## In Mesh Routing
+
+### Outbound
+
+When a pod in an ambient mesh makes an outbound request, it will be transparently redirected to the node-local ztunnel which will determine where and how to forward the request.
```

Comment:
```suggestion
When a pod in an ambient mesh makes an outbound request, it will be transparently redirected to the node-local ztunnel which will determine where and how to forward the request.
```

Question - we use `request` a lot here, but we are talking about TCP traffic here, not L7 HTTP traffic, `request` is a bit misleading.

Should  we use `traffic` instead of `request`?

Reply:
I'd say 'connection' ?